id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java

```diff
@@ -7,6 +7,7 @@
  */
 public enum ApiName {
 
-	CRDENTIALSERVICE;
+	CRDENTIALSERVICE,
+	KERNELAUDITMANAGER;

```

Comment:
There is already a property called "mosip.idrepo.audit.rest.uri",  lets try to use that so if someone uses this new change they don't need to add new properties.

Reply:
Changes done